### DISCUSSION TITLE
Creation of own window size and positions analogous to the defaults under Keyboard Shortcuts

### DISCUSSION THREAD

**Discussion by tekshrek** [rxhanson/RectanglePro-Community#286] [General]
Hello to all, one question please:

If I want to create my own window sizes and positions as predefined under Keyboard Shortcuts - how does that work please.

Examples: 
Bottom left not exactly 25% but instead 40% of the screen or ...

Centre ... not so big but already centre but a bit smaller than the default ... and so on ... 

Thank you very much.

**Comment by rxhanson** [comment]
For this you would want to use the Custom Shortcut builder. Here is a user-made tutorial on how to create custom shortcuts, with repeated commands: https://github.com/rxhanson/RectanglePro-Community/discussions/5
